zetaclient/chains/bitcoin/observer/mempool.go

```diff
@@ -0,0 +1,277 @@
+package observer
+
+import (
+	"context"
+	"time"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/pkg/errors"
+
+	"github.com/zeta-chain/node/pkg/chains"
+	"github.com/zeta-chain/node/zetaclient/chains/bitcoin/client"
+	"github.com/zeta-chain/node/zetaclient/logs"
+)
+
+const (
+	// PendingTxFeeBumpWaitBlocks is the number of blocks to await before considering a tx stuck in mempool
+	PendingTxFeeBumpWaitBlocks = 3
+
+	// PendingTxFeeBumpWaitBlocksRegnet is the number of blocks to await before considering a tx stuck in mempool in regnet
+	// Note: this is used for E2E test only
+	PendingTxFeeBumpWaitBlocksRegnet = 30
```

Comment:
I don't think these need to be exported?

Reply:
resolved